prediction/controller.py

```diff
@@ -1,15 +1,24 @@
-from prediction import independent_poisson_model
-
+from prediction import independent_poisson_model,tournament
 
 def get_current_model():
     return independent_poisson_model.IndependentPoissonModel()
 
+def get_tournament():
+    return tournament.Tournament()
 
 def make_prediction_output_currnet_model(model_outcome):
     return independent_poisson_model.make_prediction_output(model_outcome)
 
+def make_tournament_output(tournament_outcome):
+    return tournament.make_prediction_output(tournament_outcome)
```

Comment:
why does a "tournament" "make_prediction" ? 

clean code reads like a prose. it does what it says. 
When I see this code, I am confused. 

Because this function takes a tournament outcome as input and the "tournament" function weirdly makes a prediction on something that is already supposed to be an outcome.

Naming good variables is the first step to writing clean code. Please fix your variable/function/class names.